vhdbuilder/packer/test/windows-vhd-content-test.ps1

```diff
@@ -178,7 +178,7 @@ function Test-FilesToCacheOnVHD
                             continue
                         }
 
-                        Write-ErrorWithTimestamp "$mcURL is valid but the file hash is different. Expect $localFileHash but remote file hash in AzureChinaCloud is $remoteFileHash"
+                        Write-Error "$mcURL is valid but the file size is different. Expect $localFileSize but remote file size is $remoteFileSize"
```

Comment:
A bit confused about replacing "Write-ErrorWithTimestamp" with "Write-Error" here.

Reply:
Thank you. Updated.